src/subcommands/deps.rs

```diff
@@ -179,7 +179,7 @@ impl FinalfrontierApp for DepsApp {
             .transpose()?
             .unwrap();
         let cutoff = cutoff_from_matches(matches, CONTEXT_MINCOUNT, CONTEXT_TARGET_SIZE)?
-            .unwrap_or_else(|| Cutoff::MinCount(5));
+            .unwrap_or(Cutoff::MinCount(5));
```

Comment:
Pretty sure there used to be a lint against this :thinking:

Reply:
I thought so too, but now clippy wants this. I guess because `Cutoff::MinCount` is not really a function, but a trivial constructor?

Reply:
Looks like clippy hase become smarter! Semi related question, did you add dependabot to all finalfusion projects? :eyes:

Reply:
The ones that are probably still maintained. Didn't expect so many PRs ;). Somehow I hoped that the dependabot Rust plugin would've been smarter. E.g. realizing that `ndarray` and `ndarray-rand` should be updated jointly (and with that `rand`).